src/WebWorker.ts

```diff
@@ -30,12 +30,35 @@ type ExceptionThrownCallback = (
 ) => void;
 type JSHandleFactory = (obj: Protocol.Runtime.RemoteObject) => JSHandle;
 
+/**
+ * The WebWorker class represents a {@link https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API | WebWorker}.
+ * The events `workercreated` and `workerdestroyed` are emitted on the page object to signal the worker lifecycle.
+ *
+ * @remarks
```

Comment:
nitpick: I'd move the `@remarks` before the sentence about the events. The summary section (the bit before the `@remarks`) should be one short sentence at most.